OrbitGl/CaptureWindow.cpp

```diff
@@ -494,6 +494,7 @@ void CaptureWindow::Draw() {
     RenderTimeBar();
 
     Vec2 pos(m_MouseX, m_WorldTopLeftY);
+    // Green line in mouse's position
```

Comment:
Nit: in --> at

Personally, I would also remove the apostrophe and maybe clarify the dimention, so this would read:
// Vertical green line at mouse x position.

Reply:
Thank you! :)